tests/foreman/cli/test_ping.py

```diff
@@ -12,57 +12,47 @@
 
 :TestType: Functional
 
-:CaseImportance: High
+:CaseImportance: Critical
 
 :Upstream: No
 """
 import pytest
 
 from robottelo import ssh
-from robottelo.test import CLITestCase
 
 
-class PingTestCase(CLITestCase):
-    """Tests related to the hammer ping command"""
+@pytest.mark.tier1
+@pytest.mark.upgrade
+def test_positive_ping():
+    """hammer ping return code
 
-    @pytest.mark.tier1
-    @pytest.mark.upgrade
-    def test_positive_ping(self):
-        """hammer ping return code
+    :id: dfa3ab4f-a64f-4a96-8c7f-d940df22b8bf
 
-        :id: dfa3ab4f-a64f-4a96-8c7f-d940df22b8bf
+    :steps:
+        1. Execute hammer ping
+        2. Check its return code, should be 0 if all services are ok else
+           != 0
 
-        :steps:
-            1. Execute hammer ping
-            2. Check its return code, should be 0 if all services are ok else
-               != 0
+    :expectedresults: hammer ping returns a right return code
+    """
+    result = ssh.command('hammer resultping')
+    assert result.stderr.decode() == ''
```

Comment:
@mshriver this is a wrong command, it should be as `hammer ping` only :)

And add one more assert/comparison for `result.returncode` after stderr, just an additional check can be covered within the same PR.

Reply:
Its already covered further down in the test. It actually asserts the RC against what it should be given the service status.

Reply:
Yes, my bad :)